README.md

```diff
@@ -1,32 +1,95 @@
-# RebuildBlackBusiness.com
+<h1 align="center">Welcome to Rebuild Black Business 👋</h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000" />
+  <a href="https://twitter.com/ThugDebugger" target="_blank">
```

Comment:
without context, it’s unclear why this website is promoting one person’s Twitter — should that be clarified? is there a RebuildBlackBusiness Twitter account that might be more appropriate?

Reply:
Great point, I'll remove this for now until we have unified branding for Rebuild Black Business!